setup.py

```diff
@@ -45,6 +45,7 @@
     'pytest-cov',
     'pytest-attrib',
     'mock',
+    'requests-mock',
```

Comment:
It does, in the tests. But it seems that I forgot to include in the tests when I pushed. I've included them now.